pypesto/hierarchical/optimal_scaling/__init__.py

```diff
@@ -3,11 +3,12 @@
 ===============
 
 Contains the implementation of the optimal scaling approach, applied for integration
-of ordinal data in ODE modeling. Each ordinal datapoint is defined as being part of
-a category, where the mutual ordering of the categories of the same group is known.
-The category interval bounds are numerically optimized and quantitative surrogate
+of ordinal and censored data in ODE modeling. Each ordinal datapoint is defined as
+being part of a category, where the mutual ordering of the categories of the same group
+is known. The category interval bounds are numerically optimized and quantitative surrogate
 measurements are calculated to represent the ordinal measurements. This constitutes
-the inner subproblem of the hierarchical optimization problem.
+the inner subproblem of the hierarchical optimization problem. For censored data, as the
+category interval bounds are known, the surrogate measurements are directly calculated.
```

Comment:
Is the upper bound known in case of "right-censored" measurements?

Reply:
We discussed that with respect to the visualization. I'll make it nicer :) some gradient or smth :D

Reply:
I put some effort into making it nicer.
Now, in the case of right-censoring, the censoring area will have a vertical arrow going up from it with 'INF' text indicating it extends upwards.